docs/source/examples/gen_examples.py

```diff
@@ -228,8 +231,8 @@ def _gen_demonstrated_funcs(example_config_path: Path) -> dict:
     source_str = f"## Dataset source\n\nThis dataset was acquired from [{url}]({url})\n"
 
     if "openneuro" in options:
-        for key in ("include", "exclude"):
-            options[key] = options.get(key, [])
+        options["include"] = options.get("include", [])
+        options["exclude"] = options.get("exclude", [])
```

Comment:
idle curiosity: Is there any difference / advantage between this idiom and 

```suggestion
        _ = options.setdefault("include", [])
        _ = options.setdefault("exclude", [])
```

(other than the fact that `setdefault` returns the value)?

Reply:
Probably not I just never really use `setdefault` :)